homework.md

```diff
@@ -4,110 +4,112 @@
 
 What is 2 + 2?
 
-5
+Linus, 2+2 is 4, not 5, PUT ATTENTION STOP PLAYING POKEMON GO ON CLASSES!
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White, Linus are you daltonic?
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240, I need to accept I didn't even knew this.
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hypertext Markup Language (HTML) (I didn't even knew this)
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Making a website looks pretty, the hype for graphic designers and editors.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
+<h1> { Here is the section for heading elements
 
-}
+} </h1>
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+Javascript is a language that gives interface to the web pages adding interactive elements that catch the attention of the user. 
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+Github is a community of developers where ideas and works can be shared.
+Makes easy the collaborators between career colleagues.
+Team members can work on files and they can easily merge changes.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+Create your OWN copy of a project without damaging or alterate the original source. 
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+The repository is copied on local machine with the help of Git that will allow you to work on ti.
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+You're proposing changes on the document as well asking for a feedback about your collaboration. 
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+A list of the commits you made on the repo; from the most actual to the last made.
 
 ## Question 13
 
 What does it meant to "push" changes to Github?
 
-You send the changes of your local repository to Github
+Pushing means transferring your commits from your local computer to a remote repo, in this case GitHub
 
 ## Question 14
 
 What is Github?
 
-A place for us to store and share our code.
+GitHub is a code hosting platform for version control and collaboration.
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Means send your changes for evaluation of the crew or feedback, from your local computer to GitHub on the web as a remote repo.
```

Comment:
Hi Rafael,
check the difference between commit and pull requests, should help.
great job so far

Reply:
Indeed, I will make the changes on it. Thanks bud! 💯